lib/api_callable.js

```diff
@@ -35,9 +35,9 @@
 
 'use strict';
 
-var nextTick = require('process-nextick-args');
 var setTimeout = require('timers').setTimeout;
 var util = require('util');
+var process = require('process');
```

Comment:
I don't think I've seen this before. You can just access the globally-defined `process` object directly without requiring it. (@ofrobots is this a better way?)

Reply:
Agree - will change.